ansible/roles/software/dashboard/dashboard-deploy/templates/.env

```
@@ -6,3 +6,12 @@ DB_USER='{{ env_db_user }}'
 ENCRYPTION_KEY='{{ env_encryption_key }}'
 
 PRE_APPROVED_ADMINS='{{ codeigniter_pre_approved_admins | join(",") }}'
+
+S3_BUCKET=foobar
+S3_PREFIX=prefix
+USE_LOCAL_STORAGE=true
+
+PROTOCOL=http
+PROJECT_SHARED_PATH=/var/www/app
```

You'll want to template this since we install the app to a new directory on each deploy.

```suggestion
PROJECT_SHARED_PATH={{ current_source_symlink }}
```